src/core/qgscoordinatereferencesystem_p.h

```diff
@@ -138,6 +153,49 @@ class QgsCoordinateReferenceSystemPrivate : public QSharedData
     //! Whether this is a coordinate system has inverted axis
     mutable bool mAxisInverted = false;
 
+#if PROJ_VERSION_MAJOR>=6
+    mutable QReadWriteLock mProjLock;
+    mutable QMap < uintptr_t, PJ * > mProjObjects;
```

Comment:
any reason to use uintptr_t instead of PJ_CONTEXT* ?

Reply:
Nope, good point!